.github/workflows/unittest.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12"]
+        fsspec-version: ["2023.5.0", "2024.9.0"]
```

Comment:
Considering the number of final jobs of GitHub actions is the product of two matrices. So, here we only verify the min and the max fsspec versions that we plan to support to reduce the total CI time.